src/libraries/sendtohelixhelp.proj

```diff
@@ -68,6 +68,11 @@
     <HelixPreCommand Condition="'$(TargetsWindows)' != 'true' and '$(BrowserHost)' != 'windows'" Include="export MONO_ENV_OPTIONS='$(MonoEnvOptions)'" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetsWindows)' == 'true' or '$(BrowserHost)' == 'windows'">
+    <HelixPreCommand Include="taskkill.exe /f /im corerun.exe"/>
```

Comment:
Can we also add similar command for unix?

Reply:
I'd rather get this in sooner rather than later given the stability issues. I am not sure if all unix systems we have can use `pkill`, otherwise something like `ps aux | grep -i corerun | grep -v grep | awk '{ print $2 }' | xargs kill -9 ` would work.

Reply:
Makes sense. Lets get this in first.